raiden/tests/unit/test_channelstate.py

```diff
@@ -1371,7 +1376,12 @@ def test_action_close_must_change_the_channel_state():
     channel_state = create_channel_from_models(our_model1, partner_model1, partner_key1)
 
     block_number = 10
-    state_change = ActionChannelClose(canonical_identifier=channel_state.canonical_identifier)
+    state_change = ActionChannelClose(
+        canonical_identifier=channel_state.canonical_identifier,
+        participant1=our_model1.participant_address,
+        participant2=partner_model1.participant_address,
+        signed_close_tx=""
```

Comment:
I think this should be none on this case, since there is no signed close tx in a non-light channel

Can we just assign it as none or we must modifiy the core code?